bc_obps/service/data_access_service/document_service.py

```diff
@@ -24,6 +25,25 @@ def create_document(
             type=DocumentType.objects.get(name=document_type_name),
             created_by_id=user_guid,
             operation_id=operation_id,
+            status=Document.FileStatus.UNSCANNED,
         )
 
         return document
+
+    @classmethod
+    def check_document_file_status(cls, document: Document) -> Document.FileStatus:
+        file_name = document.file.name
```

Comment:
is this one the file name seen by the user? or the underlying name with the appended hash?

Reply:
This is the file name seen by the user. The Storage handler takes care of knowing the hashed filename.

Reply:
So it's both. If a filename is duplicate, it gets hashed, and in Django it's just known as that new hashed filename. This is the existing behaviour (see below, on in the Dev deployment), which we might want to address. I'll bring it up in the the teams Dev channel.

![image](https://github.com/user-attachments/assets/fc2a37d1-4fa2-4e41-bffe-3f6cedc6da3b)